cephfs/admin/metadata.go

```diff
@@ -0,0 +1,109 @@
+//go:build !(nautilus || octopus) && ceph_preview && ceph_ci_untested
+// +build !nautilus,!octopus,ceph_preview,ceph_ci_untested
+
+package admin
+
+import "C"
+
+// GetMetadata gets custom metadata on the subvolume in a volume belonging to
+// an optional subvolume group based on provided key name.
+//  PREVIEW
```

Comment:
Sorry to bother, but we don't use these PREVIEW comments anymore (since very recently). Only the `ceph_preview` tag is required. I wouldn't insist to remove it, if otherwise the PR would be ready to go, but since you have to change something anyway, please remove the "PREVIEW" comments again.

Reply:
Sure np, done now.